Tests/Integration/Components/ProductQuery/ProductQueryTest.php

```diff
@@ -13,9 +13,12 @@
 use ShopwarePlugins\Connect\Components\ProductQuery;
 use ShopwarePlugins\Connect\Components\ProductQuery\RemoteProductQuery;
 use ShopwarePlugins\Connect\Components\ProductQuery\LocalProductQuery;
+use ShopwarePlugins\Connect\Tests\DatabaseTestCaseTrait;
 
 class ProductQueryTest extends ConnectTestHelper
```

Comment:
Is it possible to remove ConnectTestHelper usage? We prefer to use traits in new tests structure.
Also DatabaseTestCaseTrait has loadFixtures method. You can use to insert something in db instead of writing queries in your tests. If possible of course.